opengb/seed-core
# FLEX-263: :bug: Fix version conflict with Python packages shapely and numpy in SEED Docker images

#### Any background context you want to provide?
See https://github.com/SEED-platform/seed/pull/4696 in the upstream GitHub repo.

#### What's this PR do?
- :bug: Fixes inability to start SEED Docker images due to a version conflict involving the Python packages `shapely` and `numpy`.

#### How should this be manually tested?
1. Checkout this branch locally.
2. Run `docker build .` to build a new Docker image from this version of the code.
3. Try running the newly-built SEED Docker image.
4. You shouldn't see any errors related to `shapely` or `numpy` while SEED is starting up.

#### What are the relevant tickets?
Encountered while working on [FLEX-263](https://opengb.atlassian.net/browse/FLEX-263)

#### Screenshots (if appropriate)
N/A

[FLEX-263]: https://opengb.atlassian.net/browse/FLEX-263?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
